ui.config/src/main/content/jcr_root/apps/venia/osgiconfig/config/com.adobe.cq.commerce.graphql.client.impl.GraphqlClientImpl~default.cfg.json

```diff
@@ -6,5 +6,10 @@
   "socketTimeout": 5000,
   "maxHttpConnections": 20,
   "requestPoolTimeout": 2000,
-  "acceptSelfSignedCertificates": false
+  "acceptSelfSignedCertificates": false,
+  "cacheConfigurations": [
+    "venia/components/commerce/navigation:true:1:300",
```

Comment:
This should be more the 1, maybe 5 for the beginning since multiple stores can be used which might have different nav trees

Reply:
Good point. I've fixed it.